src/components/timeline/FullTimeline.js

```diff
@@ -167,24 +166,30 @@ class TimelineSettingsHiddenTracks extends React.PureComponent<{|
 
 class TimelineSettingsActiveTabView extends React.PureComponent<{|
   +activeBrowsingContextID: BrowsingContextID | null,
-  +showTabOnly: BrowsingContextID | null,
-  +changeViewAndRecomputeProfileData: typeof changeViewAndRecomputeProfileData,
+  +timelineTrackOrganization: TimelineTrackOrganization,
+  +changeTimelineTrackOrganization: typeof changeTimelineTrackOrganization,
 |}> {
   _toggleShowTabOnly = () => {
```

Comment:
nit: It would be good to rename this function to `toggleActiveTabView` or something like this now too since we don't have `showTabOnly` state anymore.